source/event.c

```diff
@@ -431,9 +431,10 @@ static void async_run_gpe_handler(uacpi_handle opaque)
             break;
         }
 
+        uacpi_object_name name = uacpi_namespace_node_name(event->aml_handler);
```

Comment:
I try to keep variable declarations at the top of the current block, so would be nice if it was
```diff
 uacpi_object *method_obj
+ uacpi_object_name name;

...

+ name = uacpi_namespace_node_name(event->aml_handler);
uacpi_trace(...)
```